### ISSUE TITLE
Traffic light direction is wrong in Apollo map

### ISSUE THREAD

**Issue by qiongwu1003** [lgsvl/simulator#1143]
Hi,

simulator : 202006
Apollo : 5.0

I made a simple 3D map and HD map reference [here](https://www.lgsvlsimulator.com/docs/map-annotation/#create-traffic-signals), it shows well in the simulator. But when I generate the HD map to Apollo according to this [issue](https://github.com/lgsvl/simulator/issues/370), some traffic lights directions are wrong. Am I did something wrong?

Below is the one error traffic light and its parameter.
![simulator](https://user-images.githubusercontent.com/65001214/100031088-bcc06c80-2e2f-11eb-88f7-5c46e7dc5cde.png)
![apollo](https://user-images.githubusercontent.com/65001214/100031097-c0ec8a00-2e2f-11eb-8fd6-3beab686c68b.png)
![all](https://user-images.githubusercontent.com/65001214/100031108-c6e26b00-2e2f-11eb-92a3-122eb5441abe.png)
![stoplinedirection](https://user-images.githubusercontent.com/65001214/100031116-cb0e8880-2e2f-11eb-90df-67ca83c7fddc.png)

**Comment by luqiang21** [comment]
@qiongwu1003 Can you show us the annotations of the signal?

**Comment by qiongwu1003** [comment]
Do you mean this one:
![signalannotation](https://user-images.githubusercontent.com/65001214/100034858-0d3bc800-2e38-11eb-8ba5-062c19252243.png)
![signalannotation1](https://user-images.githubusercontent.com/65001214/100034862-0dd45e80-2e38-11eb-8367-dcc6a2b6805a.png)

**Comment by luqiang21** [comment]
@qiongwu1003 Yes, are your signals having the same directions as these annotations?

**Comment by qiongwu1003** [comment]
Yes, they are. Z axis forward and Y axis up.
![signals](https://user-images.githubusercontent.com/65001214/100052046-13da3780-2e58-11eb-9c78-820db99caf18.png)

**Comment by EricBoiseLGSVL** [comment]
@qiongwu1003 Can you please screen shot the intersection so we can see if the signal and stopline objects are facing the correct way?  Be sure Toggle Tool Handle Rotation is in Local Space

Like so
![SignalHelp00](https://user-images.githubusercontent.com/45106230/100288388-6cc6d000-2f2b-11eb-8c1a-f6ec3c50ccbe.png)
![SignalHelp01](https://user-images.githubusercontent.com/45106230/100288398-72241a80-2f2b-11eb-8c2e-215face8120b.png)

**Comment by qiongwu1003** [comment]
@EricBoiseLGSVL 
Thanks for your reply, below are the screenshot.
![signal](https://user-images.githubusercontent.com/65001214/100292821-8b28dc00-2fbc-11eb-9987-4c9b099f3edb.png)
![stop](https://user-images.githubusercontent.com/65001214/100292827-8ebc6300-2fbc-11eb-8112-ba3d62f5edf2.png)

**Comment by EricBoiseLGSVL** [comment]
Hmm, that is right,  Very odd that the signals are inverted.

**Comment by qiongwu1003** [comment]
@EricBoiseLGSVL 
I also found that the traffic lights' directions are wrong in some intersections of the Sanfrancisco map.
Maybe the same error with my HD map? Or my computer's problem, setting or something? Thanks a lot.
![Sanfan](https://user-images.githubusercontent.com/65001214/100294618-5029a700-2fc2-11eb-9e97-f79b210b4460.png)

**Comment by EricBoiseLGSVL** [comment]
Can you screen shot the intersection please.  It will help to debug both issues

**Comment by qiongwu1003** [comment]
It is “MapIntersection_junction_89_0” in Sanfrancisco HD map.
The "MapSignal_signal_124" is wrong.
![124_error](https://user-images.githubusercontent.com/65001214/100314460-a19d5a80-2ff1-11eb-8451-92359bb8e2bd.png)

The "MapSignal_signal_122" is correct, but I cant find out the difference.
![122](https://user-images.githubusercontent.com/65001214/100314512-c4c80a00-2ff1-11eb-9ab3-369626f52e52.png)

**Comment by EricBoiseLGSVL** [comment]
Thanks for posting.  These all look correct. Maybe this is an issue with exporting the HDmap.  I'll ping @luqiang21 and see if he knows how to check the export.
